nvflare/fuel/f3/cellnet/cell.py

```diff
@@ -1112,7 +1073,7 @@ def _send_target_messages(
                 self.logger.debug(f"{self.my_info.fqcn}: invoking outgoing request filters")
                 assert isinstance(req_filters, list)
                 for f in req_filters:
-                    assert isinstance(f, _CB)
+                    assert isinstance(f, Callback)
```

Comment:
If we need this check on production code, we need to change assert to ```if not isinstance(_cb, Callback): raise ValueError```

Reply:
Same as above.